README.rst

```diff
@@ -160,6 +160,8 @@ the loop, because `late-binding closures are a classic gotcha
 This check identifies exception types that are specified in multiple ``except``
 clauses. The first specification is the only one ever considered, so all others can be removed.
 
+**B026**: Argument unpacking after keyword argument.
```

Comment:
```suggestion
**B026**: Argument unpacking after keyword argument is strongly discouraged. This behavior is uncommon and can lead to unnecessary confusion when reading the code thus making it prone to causing bugs. There was cpython discussion of disallowing but legacy usage and parser limitations make it difficult.
```
+1 on more description here. A suggestion that I'm more than happy to have reworded + made better.

Reply:
```suggestion
**B026**: Star-arg unpacking after a keyword argument is strongly discouraged, because
it only works when the keyword parameter is declared after all parameters supplied by 
the unpacked sequence, and this change of ordering can surprise and mislead readers.
```

I think this is compact enough to also use as the runtime message, and explains specifically what we think is wrong (the mismatched order can be misleading) rather than non-actionable historical details.

I do like the historical details and I'd be happy to keep "There was [cpython discussion of disallowing](https://github.com/python/cpython/issues/82741) this syntax, but legacy usage and parser limitations make it difficult." with that link in the readme too, but not print it at runtime.

Reply:
I'm not super happy about the wording and needed to read it multiple times to understand that it indeed correctly explains the issue. Still, I have no better way so what gives. This is really hard to explain without an example TBH.

+1 on keeping the historical details to the README.